### ISSUE TITLE
Make error in building the current version of Drasil

### ISSUE THREAD

**Issue by oluowoj** [JacquesCarette/Drasil#1085]
@JacquesCarette,

Any thoughts on how I can resolve this issue. I ran make to build the current version of Drasil but I get the error below. See the orange arrow.
 Also see the green arrow, I expected to see a build folder but it looks like the executables are in the location the green arrow is pointing to. 

<img width="427" alt="image" src="https://user-images.githubusercontent.com/43192745/50544360-72410180-0bc0-11e9-9024-416db5362cbc.png">

I also have make installed:

<img width="660" alt="image" src="https://user-images.githubusercontent.com/43192745/50544367-ae746200-0bc0-11e9-9f60-4b44871610cc.png">

adding @vajihehm , looks like she is experiencing the same.

**Comment by oluowoj** [comment]
Thanks, no luck yet but will keep trying

**Comment by vajihehm** [comment]
@oluowoj  Thank you Olu for creating issue and explaining whatever we did to solve the problem. Just one update in step 2, It would be better to put the following link for 'Make for Windows' that it has a description for the 'Make tool for windows' and there is required packages to download as well:
http://gnuwin32.sourceforge.net/packages/make.htm
@JacquesCarette As Olu mentioned the problem was solved with those steps. However, in step 3 the command of 'chcp.cpm 65001' set the encoding of Terminal to UTF-8.  Before the execution of this command, we had error which we reported. I am really curious to know why is that required  to set the encoding of Terminal to UTF-8 to build and execution of Drasil?
